src/pages/Login/Login.jsx

```diff
@@ -0,0 +1,84 @@
+import { BsArrowRight } from "react-icons/bs";
+import { Link, useNavigate } from "react-router-dom";
+import "./Login.css";
+import { getUser } from "../../services";
+import { useUser } from "../../context";
+export const Login = () => {
+  const { setUser } = useUser();
+  const navigator = useNavigate();
+
+  const formHandler = (e) => {
+    e.preventDefault();
+    const [email, password] = e.target.elements;
+
+    (async () => {
+      try {
+        const { data, status } = await getUser(email.value, password.value);
+        if (status === 200) {
+          localStorage.setItem("token", JSON.stringify(data.encodedToken));
+          setUser({ ...data.foundUser, isLoggedIn: true });
+          navigator("/products", { replace: true });
+        } else console.error("EMAIL OR PASSWORD IS INCORRECT", data);
+      } catch (err) {
+        console.log("FAILED TO UPDATE USER", err);
+      }
```

Comment:
You have added a `try{} catch(e){} ` in `getUser()`  so you can remove it from here

Reply:
Done.